pkg/reconciler/v1alpha1/broker/broker_test.go

```diff
@@ -17,27 +17,25 @@ limitations under the License.
 package broker
 
 import (
-	"context"
-	"errors"
 	"fmt"
-	"strings"
 	"testing"
 
-	"github.com/google/go-cmp/cmp"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"k8s.io/apimachinery/pkg/util/intstr"
+
 	"github.com/knative/eventing/pkg/apis/eventing/v1alpha1"
-	controllertesting "github.com/knative/eventing/pkg/reconciler/testing"
+	"github.com/knative/eventing/pkg/reconciler"
+	testing2 "github.com/knative/eventing/pkg/reconciler/testing"
```

Comment:
change this to `. "github.com/knative/eventing/pkg/reconciler/testing"`

Reply:
done, thanks!